nucliadb/nucliadb/writer/api/v1/upload.py

```diff
@@ -754,3 +755,12 @@ def maybe_b64decode(some_string: str) -> str:
     except ValueError:
         # not b64encoded
         return some_string
+
+
+def guess_content_type(filename: str) -> str:
+    default_ct = "application/octet-stream"
```

Comment:
NIT: I think `_ct` suffix doesn't bring value here. As it's a short function with a clear name, it's easy to understand `default` as default content type